docs/Velox.md

```diff
@@ -15,17 +16,51 @@ export JAVA_HOME=path/to/java/home
 export PATH=$JAVA_HOME/bin:$PATH
 ```
 
-# 2 Build Velox Jar
+# 2 Build Gluten with Velox Backend
 
-Since the mvn script calls setup-ubuntu.sh to install dependency libraries, so we need to run it from <b>root</b> group.
+```shell script
 
-The command below clones velox source code from [OAP-project/velox](https://github.com/oap-project/velox) to ep/build-velox/build/velox_ep. Then it applies some patches to Velox build script and builds the velox library.
+## install gcc and libraries to build arrow
+apt-get update && apt-get install -y sudo locales wget tar tzdata git ccache cmake ninja-build build-essential llvm-11-dev clang-11 libiberty-dev libdwarf-dev libre2-dev libz-dev libssl-dev libboost-all-dev libcurl4-openssl-dev openjdk-8-jdk maven
 
-```shell script
+## make sure jdk8 is used
+export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+export PATH=$JAVA_HOME/bin:$PATH
+
+##setup proxy if needed
+export http_proxy=xxxx
+export https_proxy=xxxx
+
+##config maven proxy
+mkdir ~/.m2/
+apt install vim
+vim ~/.m2/settings.xml
+
+git clone https://github.com/oap-project/gluten.git
+
+
+## fetch arrow and compile
+cd /path_to_gluten/ep/build-arrow/src/
+./get_arrow.sh
+./build_arrow_for_velox.sh
+
+## fetch velox and compile
+cd /path_to_gluten/ep/build-velox/src/
```

Comment:
for “## fetch velox and compile”,
is compiling in ./get_velox.sh or ./build_velox.sh?

Reply:
./get_velox.sh -> fetch velox
./build_velox.sh ->compile